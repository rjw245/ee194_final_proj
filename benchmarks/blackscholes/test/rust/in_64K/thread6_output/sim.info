{'cmdline': ['/h/hzhou01/comp140/sniper/run-sniper',
             '-d',
             './thread6_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '6',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell24',
 'pin_version': 71313,
 'rusage': (53944.849136,
            620.726635,
            173012,
            0,
            0,
            0,
            442607,
            0,
            0,
            0,
            12384,
            0,
            0,
            0,
            41927028,
            141401),
 'snipercmd': ['bash',
               '-c',
               '/h/hzhou01/comp140/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/hzhou01/comp140/sniper/lib/pin_sim -c /h/hzhou01/comp140/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/hzhou01/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread6_output --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg --config=/h/hzhou01/comp140/sniper/config/gainestown.cfg --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 6 ../../../inputs/in_64K.txt'],
 't_elapsed': 23255.930671930313,
 't_start': 1462232497.535089,
 'tracecmds': [],
 'user': 'hzhou01',
 'vmem': 3944411136L}
