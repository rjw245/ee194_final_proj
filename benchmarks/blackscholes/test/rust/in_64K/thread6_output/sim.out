                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |  4219244415 |  2218008603 |  2218024948 |  2218050441 |  2218013475 |  2218012261 |  2218047956 |           0
  Cycles                             |  4365500762 |  4365500762 |  4365500762 |  4365500762 |  4365500762 |  4365500762 |  4365500762 |  4365500762
  IPC                                |        0.97 |        0.51 |        0.51 |        0.51 |        0.51 |        0.51 |        0.51 |        0.00
  Time (ns)                          |  1641165700 |  1641165700 |  1641165700 |  1641165700 |  1641165700 |  1641165700 |  1641165700 |  1641165700
  Idle time (ns)                     |   707124682 |   936370620 |   943557246 |   944296004 |   935548223 |   943326447 |   934045186 |  1641165700
  Idle time (%)                      |       43.1% |       57.1% |       57.5% |       57.5% |       57.0% |       57.5% |       56.9% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   148261973 |    78336016 |    78451287 |    78461410 |    78365968 |    78450606 |    78357456 |           0
  num incorrect                      |     5881055 |     2136298 |     2020291 |     2011963 |     2105077 |     2020782 |     2114785 |           0
  misprediction rate                 |       3.82% |       2.65% |       2.51% |       2.50% |       2.62% |       2.51% |       2.63% |       0.00%
  mpki                               |        1.39 |        0.96 |        0.91 |        0.91 |        0.95 |        0.91 |        0.95 |        0.00
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   903862237 |   364104015 |   364104531 |   364109818 |   364104812 |   364104042 |   364107894 |           0
    num misses                       |         195 |          75 |          71 |          76 |          71 |          71 |          71 |           0
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |  2813501862 |  1275201520 |  1275222766 |  1275230200 |  1275207845 |  1275209277 |  1275234045 |           0
    num misses                       |      219602 |        5083 |        5519 |        5945 |        5515 |        5518 |        5517 |           0
    miss rate                        |       0.01% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.05 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |      219797 |        5158 |        5590 |        6021 |        5586 |        5589 |        5588 |           0
    num misses                       |         814 |         263 |         215 |         257 |         214 |         224 |         217 |           0
    miss rate                        |       0.37% |       5.10% |       3.85% |       4.27% |       3.83% |       4.01% |       3.88% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   903862237 |   364104015 |   364104531 |   364109818 |   364104812 |   364104042 |   364107894 |           0
    num cache misses                 |    22987561 |       12812 |       12750 |       12909 |       12720 |       12763 |       12756 |           0
    miss rate                        |       2.54% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |        inf%
    mpki                             |        5.45 |        0.01 |        0.01 |        0.01 |        0.01 |        0.01 |        0.01 |         inf
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |  2813502564 |  1275201762 |  1275223037 |  1275230486 |  1275208141 |  1275209580 |  1275234361 |           0
    num cache misses                 |      103140 |      479427 |      479190 |      478558 |      478681 |      478245 |      478476 |           0
    miss rate                        |       0.00% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |        inf%
    mpki                             |        0.02 |        0.22 |        0.22 |        0.22 |        0.22 |        0.22 |        0.22 |         inf
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |    23090886 |      492259 |      491941 |      491479 |      491447 |      491034 |      491245 |           0
    num cache misses                 |       39670 |      300668 |      290282 |      290450 |      299574 |      301259 |      298402 |           0
    miss rate                        |       0.17% |      61.08% |      59.01% |      59.10% |      60.96% |      61.35% |      60.74% |        inf%
    mpki                             |        0.01 |        0.14 |        0.13 |        0.13 |        0.14 |        0.14 |        0.13 |         inf
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |       39867 |      300688 |      290283 |      290462 |      299620 |      301285 |      298415 |           0
    num cache misses                 |       33908 |         771 |         383 |         537 |        5351 |        4906 |        4616 |           0
    miss rate                        |      85.05% |       0.26% |       0.13% |       0.18% |       1.79% |       1.63% |       1.55% |        inf%
    mpki                             |        0.01 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |         inf
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       18257 |           0 |           0 |           0 |       18258 |           0 |           0 |           0
  average dram access latency (ns)   |       58.32 |         inf |         inf |         inf |       58.51 |         inf |         inf |         inf
  average dram queueing delay        |        4.89 |        0.00 |        0.00 |        0.00 |        5.08 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |        3899 |         180 |          65 |          44 |          16 |          50 |          51 |           0
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |          30 |          72 |           9 |          23 |        4363 |        4212 |        4168 |           0
